CruCentralCoast/Community/CommunityGroups/CommunityGroupCell.swift

```diff
@@ -24,14 +24,10 @@ class CommunityGroupCell: UITableViewCell {
     override func awakeFromNib() {
         super.awakeFromNib()
         
-        self.cellMask.layer.cornerRadius = 20
-        self.cellMask.layer.shadowColor = UIColor.black.cgColor
-        self.cellMask.layer.shadowOpacity = 0.2
-        self.cellMask.layer.shadowOffset = CGSize.zero
-        self.cellMask.layer.shadowRadius = 10
-        self.bannerImageView.layer.cornerRadius = 20
-        self.bannerImageView.layer.maskedCorners = [.layerMaxXMinYCorner, .layerMinXMinYCorner]
-        self.bannerImageView.layer.masksToBounds = true
+
+//        self.bannerImageView.layer.cornerRadius = 20
+//        self.bannerImageView.layer.maskedCorners = [.layerMaxXMinYCorner, .layerMinXMinYCorner]
+//        self.bannerImageView.layer.masksToBounds = true
```

Comment:
general feedback, here and elsewhere in your files changed, I'd remove any commented out code you have. if it's something you'll use later for something mark it with a TODO or something